sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClient.cs

```diff
@@ -481,6 +510,18 @@ private static CreateCallRequestInternal CreateCallRequest(CreateCallOptions opt
             return request;
         }
 
+        /// <summary>
+        /// Validates an Https Uri.
+        /// </summary>
+        /// <param name="uri"></param>
+        /// <returns></returns>
+        public static bool IsValidHttpsUri(Uri uri) {
```

Comment:
Lets not expose this function - make it either internal or private

Reply:
changed to private. ty!